### ISSUE TITLE
Using pry 0.13.0 results in the error NoMethodError: undefined method `theme_options' for #<Pry::Config:0x000055dbe462bdd8>

### ISSUE THREAD

**Issue by YorickPeterse** [kyrylo/pry-theme#59]
Full error:

```
when_started hook failed: NoMethodError: undefined method `theme_options' for #<Pry::Config:0x000055dbe462bdd8>
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/config.rb:250:in `method_missing'
(see pry_instance.hooks.errors to debug)
2.7.1p83 > error = pry_instance.hooks.errors.first
=> #<NoMethodError: undefined method `theme_options' for #<Pry::Config:0x000055dbe462bdd8>>
2.7.1p83 > puts error.backtrace
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/config.rb:250:in `method_missing'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-theme-1.3.0/lib/pry-theme/when_started_hook.rb:22:in `apply_config'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-theme-1.3.0/lib/pry-theme/when_started_hook.rb:16:in `call'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/hooks.rb:111:in `block in exec_hook'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/hooks.rb:109:in `map'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/hooks.rb:109:in `exec_hook'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/pry_instance.rb:396:in `exec_hook'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/pry_instance.rb:95:in `initialize'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/repl.rb:15:in `new'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/repl.rb:15:in `start'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/pry_class.rb:191:in `start'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/lib/pry/cli.rb:119:in `start'
/home/yorickpeterse/.gem/ruby/2.7.1/gems/pry-0.13.0/bin/pry:13:in `<top (required)>'
/home/yorickpeterse/.gem/ruby/2.7.1/bin/pry:23:in `load'
/home/yorickpeterse/.gem/ruby/2.7.1/bin/pry:23:in `<main>'
```

This is using the following config:

```
Pry.config.theme = 'happy_hacking'

Pry.config.prompt = Pry::Prompt.new(
  'custom',
  'Custom Pry prompt',
  [
  # Regular prompt.
  proc do |obj, nest_level, _|
    "#{RUBY_VERSION}p#{RUBY_PATCHLEVEL} > "
  end,
  # Wait prompt
  proc do |obj, nest_level, _|
    "#{RUBY_VERSION}p#{RUBY_PATCHLEVEL} * "
  end
  ]
)
```

**Comment by kyrylo** [comment]
Wow, you still use this gem 😄
I'll look into that.

**Comment by YorickPeterse** [comment]
@kyrylo As long as the default colors of Pry remaing somewhat unpleasant, I will keep using this Gem :smile:

**Comment by kyrylo** [comment]
Fixed by https://github.com/pry/pry/pull/2126 in Pry and released in Pry v0.13.1. 
Please update: https://rubygems.org/gems/pry/versions/0.13.1

**Comment by YorickPeterse** [comment]
@kyrylo Thanks! :fireworks:
